src/main/java/Duke.java

```diff
@@ -1,10 +1,151 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
 public class Duke {
+    private static final String LINE = "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~";
+    private static final Scanner input = new Scanner(System.in);
+
+    private static final String COMMAND_BYE = "bye";
+    private static final String COMMAND_LIST = "list";
+    private static final String COMMAND_DONE = "done";
+    private static final String COMMAND_TODO = "todo";
+    private static final String COMMAND_DEADLINE = "deadline";
+    private static final String COMMAND_EVENT = "event";
+
+    private static final List<Task> tasks = new ArrayList<Task>();
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        printWelcomeMessage();
+        String userInput;
+         do {
+            System.out.println(LINE);
+            userInput = input.nextLine().trim();
+            System.out.println(LINE);
+        } while(executeUserCommand(userInput));
+    }
+
+    /**
+     * Prints welcome message for the bot.
+     */
+    private static void printWelcomeMessage() {
+        System.out.println(LINE);
+        System.out.println("                Hewwo! I'm UwU Bot\n"
+                          +"              Hwere to hwelp you out");
```

Comment:
Consider changing the indentation here to follow the coding standards regarding wrapped lines (i.e. "Indentation for wrapped lines should be 8 spaces more than the parent line.").

Reply:
Okie, thanks for spotting!